nestkernel/model_manager.h

```diff
@@ -132,12 +133,7 @@ class ModelManager : public ManagerInterface
    * @param name The name under which the ConnectorModel will be registered.
    */
   template < template < typename targetidentifierT > class ConnectionT >
-  void register_connection_model( const std::string& name,
-    const RegisterConnectionModelFlags flags = default_connection_model_flags );
-
-  template < template < typename targetidentifierT > class ConnectionT >
-  void register_secondary_connection_model( const std::string& name,
-    const RegisterConnectionModelFlags flags = default_secondary_connection_model_flags );
+  void register_connection_model( const std::string& name );
```

Comment:
After simplifying the declaration of this function, wouldn't it be even better to discard the `string name` argument, and use a macro to convert the class name to a string inside the definition of `register_connection_model`.

Reply:
I remember debugging situations where it came in handy that I was able to register the same model under different names and would thus keep this flexible for now.